### PR TITLE
Windows portability issues

### PR DESCRIPTION
**Pull request by wattdave**
Most of these are minor and trivial, but expecting the HOME environment
variable to exist seemed important to fix. On Windows versions back to
Windows XP, 'USERPROFILE' exists by default and points to the user's home
folder.

http://ss64.com/nt/syntax-variables.html

### CODE REVIEW

File: aomi/cli.py [hunk 1]
```diff
@@ -47,15 +47,15 @@ def parser_factory(operation):
         parser.add_option('--secrets',
                           dest='secrets',
                           help='Path where secrets are stored',
-                          default="%s/.secrets" % os.getcwd())
+                          default=os.path.join(os.getcwd(), ".secrets"))
         parser.add_option('--policies',
                           dest='policies',
                           help='Path where policies are stored',
-                          default="%s/vault" % os.getcwd())
```
Comment (wattdave) [inline aomi/cli.py]: Is this intended to be a folder? If so, then os.path.join(os.getcwd(), "vault", "") is more correct.

Comment (otakup0pe) [inline aomi/cli.py]: Yes, this is a folder...

Comment (wattdave) [inline aomi/cli.py]: Okay, I can submit it with the code change I mentioned. The effect, though, will be that there will be a trailing slash at the end of that default, *unlike* what's there now. Should I leave it?

Comment (otakup0pe) [inline aomi/cli.py]: I think that's OK, since this should always be a directory?

Comment (wattdave) [inline aomi/cli.py]: Okay, updating that. Also added some use of os.sep.

**Pull request merged**